packages/admin/admin/src/form/Field.tsx

```diff
@@ -111,7 +111,6 @@ export function Field<FieldValue = any, FieldElement extends HTMLElement = HTMLE
                     subscription={{ values: true }}
                     onChange={async ({ values }) => {
                         if (!setFieldData) {
-                            // eslint-disable-next-line no-console
```

Comment:
Why can this be removed? Is it because it's a warning, or does the rule no longer work? IMO we should prevent console.log in the Admin and site